coq-metacoq-checker.opam

```
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+version: "2.2"
+maintainer: "matthieu.sozeau@inria.fr"
+homepage: "https://metacoq.github.io/metacoq"
+dev-repo: "git+https://github.com/MetaCoq/metacoq.git#coq-8.8"
+bug-reports: "https://github.com/MetaCoq/metacoq/issues"
+authors: ["Abhishek Anand <aa755@cs.cornell.edu>"
+          "Simon Boulier <simon.boulier@inria.fr>"
+          "Cyril Cohen <cyril.cohen@inria.fr>"
+          "Yannick Forster <forster@ps.uni-saarland.de>"
+          "Fabian Kunze <fkunze@fakusb.de>"
+          "Gregory Malecha <gmalecha@gmail.com>"
+          "Matthieu Sozeau <matthieu.sozeau@inria.fr>"
+          "Nicolas Tabareau <nicolas.tabareau@inria.fr>"
+          "Théo Winterhalter <theo.winterhalter@inria.fr>"
+]
+license: "MIT"
+build: [
+  [make "-j%{jobs}%" "template-coq"]
+  [make "-j%{jobs}%" "checker"]
+]
+install: [
+  [make "-C" "checker" "install"]
+]
+depends: [
+  "ocaml" {> "4.02.3"}
+  "coq" {>= "8.8" & < "8.9~"}
+  "coq-metacoq-template" {= "2.2"}
```

```suggestion
  "coq-metacoq-template" {= version}
```
To make copy-paste easier, we recommend using `version` here to depend on the "same" versions of packages released together.